jck/README.md

```diff
@@ -68,3 +68,27 @@ locally within Docker container.
    initialize the JCK docker image.
 
 4. Once you are in the Docker container, follow [How-to Run customized JCK test targets](#how-to-run-customized-jck-test-targets) or [How-to Run JCK Tests](#how-to-run-jck-tests) to run JCK tests locally.
+
+A quick start to run JCK test in docker image
+
+```
+// build docker image and run it
+cd <openjdk-tests_root>/buildenv/docker
+docker build -t openjdk-test .
+docker run -it -v <path_to_openjdk-tests_root>:/test  -v <JCK_MATERIAL_ROOT>:/jck_root openjdk-test /bin/bash
```

Comment:
- should we provide cmd for git clone for openjdk-tests?
- Is `JCK_MATERIAL_ROOT` a special variable? If not, I would prefer we keep it lower case, similar to the rest (i.e., `path_to_openjdk-tests_root`)

Reply:
updated the change set to include cmd for git clone for openjdk-tests and change `JCK_MATERIAL_ROOT` to lower case